tuf/src/repository/ephemeral.rs

```diff
@@ -32,6 +32,15 @@ where
             _interchange: PhantomData,
         }
     }
+
+    /// Returns a [EphemeralTransaction] for manipulating this repository. This allows callers to
+    /// stage a number of mutations, and optionally commit them all at once.
+    pub fn transaction(&mut self) -> EphemeralTransaction<'_, D> {
```

Comment:
I'd name this something else if it doesn't have database-like transaction semantics, in particular that partially-applied changes are rolled back on failure. Perhaps `batch_update`?

Reply:
Renamed to `batch_update()`.